bubblewrap.c

```diff
@@ -305,74 +308,75 @@ usage (int ecode, FILE *out)
+           "    --overlay-src SRC             Read files from SRC in the following overlay\n"
+           "    --overlay RWSRC WORKDIR DEST  Mount overlayfs on DEST, with RWSRC as the host path for writes and\n"
+           "                                  WORKDIR an empty directory on the same filesystem as RWSRC\n"
+           "    --tmp-overlay DEST            Mount overlayfs on DEST, with writes going to an invisible tmpfs\n"
+           "    --ro-overlay DEST             Mount overlayfs read-only on DEST\n"
+           "    --exec-label LABEL            Exec label for the sandbox\n"
+           "    --file-label LABEL            File label for temporary sandbox content\n"
+           "    --proc DEST                   Mount new procfs on DEST\n"
+           "    --dev DEST                    Mount new dev on DEST\n"
+           "    --tmpfs DEST                  Mount new tmpfs on DEST\n"
+           "    --mqueue DEST                 Mount new mqueue on DEST\n"
+           "    --dir DEST                    Create dir at DEST\n"
+           "    --file FD DEST                Copy from FD to destination DEST\n"
+           "    --bind-data FD DEST           Copy from FD to file which is bind-mounted on DEST\n"
+           "    --ro-bind-data FD DEST        Copy from FD to file which is readonly bind-mounted on DEST\n"
+           "    --symlink SRC DEST            Create symlink at DEST with target SRC\n"
+           "    --seccomp FD                  Load and use seccomp rules from FD (not repeatable)\n"
+           "    --add-seccomp-fd FD           Load and use seccomp rules from FD (repeatable)\n"
+           "    --block-fd FD                 Block on FD until some data to read is available\n"
+           "    --userns-block-fd FD          Block on FD until the user namespace is ready\n"
+           "    --info-fd FD                  Write information about the running container to FD\n"
+           "    --json-status-fd FD           Write container status to FD as multiple JSON documents\n"
+           "    --new-session                 Create a new terminal session\n"
+           "    --die-with-parent             Kills with SIGKILL child process (COMMAND) when bwrap or bwrap's parent dies.\n"
+           "    --as-pid-1                    Do not install a reaper process with PID=1\n"
+           "    --cap-add CAP                 Add cap CAP when running as privileged user\n"
+           "    --cap-drop CAP                Drop cap CAP when running as privileged user\n"
+           "    --perms OCTAL                 Set permissions of next argument (--bind-data, --file, etc.)\n"
+           "    --size BYTES                  Set size of next argument (only for --tmpfs)\n"
+           "    --chmod OCTAL PATH            Change permissions of PATH (must already exist)\n"
+           "    --scope-abstract-unix-sockets Scope access to abstract unix sockets to within in the sandbox\n"
```

Comment:
For this initial review pass I've assumed that you didn't change the `--help` except for re-indenting. I'm leaving this comment as a reminder that someone still needs to check this.

Reply:
Should be a non-issue now that we've changed the option name to be shorter.